src/v2/Apps/Categories/CategoriesApp.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { MetaTags } from "v2/Components/MetaTags"
+import { CategoriesIntro } from "./Components/CategoriesIntro"
+import { GeneFamiliesFragmentContainer } from "./Components/GeneFamilies"
+import { CategoriesApp_geneFamiliesConnection } from "v2/__generated__/CategoriesApp_geneFamiliesConnection.graphql"
+import { StickyNavFragmentContainer } from "./Components/StickyNav"
+import { DROP_SHADOW, FullBleed, Spacer } from "@artsy/palette"
+import { StickyProvider, Sticky } from "v2/Components/Sticky"
+import { AppContainer } from "../Components/AppContainer"
+
+interface CategoriesAppProps {
+  geneFamiliesConnection: CategoriesApp_geneFamiliesConnection
+}
+
+const CategoriesApp: React.FC<CategoriesAppProps> = props => {
+  const { geneFamiliesConnection } = props
+  return (
+    <>
+      <MetaTags pathname="categories" />
+      <Spacer mt={[2, 4]} />
+      <CategoriesIntro />
+      <StickyProvider>
+        <Sticky>
+          {({ stuck }) => {
+            return (
+              <FullBleed style={stuck ? { boxShadow: DROP_SHADOW } : undefined}>
+                <AppContainer maxWidth="none">
+                  <StickyNavFragmentContainer
```

Comment:
This should be left aligned with the app container to start with; it doesn't look that way from the screenshot?

Reply:
oop, that's just poor screenshotting on my part - I had already scrolled the rail. It's correctly left-aligned with the app container on page load